components/apps_form/apps_form_field/apps_form_select_field.tsx

```diff
@@ -127,6 +205,10 @@ export default class AppsFormSelectField extends React.PureComponent<Props, Stat
             selectComponent = this.renderDynamicSelect();
         } else if (field.type === 'static_select') {
```

Comment:
Can we use constants `AppFieldTypes.STATIC_SELECT` and `AppFieldTypes.DYNAMIC_SELECT` in this file?